From e147bdce0751d44d1bb8506da192f939000ec913 Mon Sep 17 00:00:00 2001
From: Patrick Gaskin <patrick@pgaskin.net>
Date: Sat, 4 Dec 2021 20:19:07 -0500
Subject: [PATCH] sepolicy: fix moto health for sm6125

---
 common/dynamic/hwservice_contexts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/common/dynamic/hwservice_contexts b/common/dynamic/hwservice_contexts
index b0206e1..7d5b4c1 100644
--- a/common/dynamic/hwservice_contexts
+++ b/common/dynamic/hwservice_contexts
@@ -1,4 +1,3 @@
-motorola.hardware.health::IMotHealth                                    u:object_r:hal_health_hwservice:s0
 vendor.lineage.biometrics.fingerprint.inscreen::IFingerprintInscreen    u:object_r:hal_lineage_fod_hwservice:s0
 vendor.lineage.camera.motor::ICameraMotor                               u:object_r:hal_lineage_camera_motor_hwservice:s0
 vendor.lineage.fastcharge::IFastCharge                                  u:object_r:hal_lineage_fastcharge_hwservice:s0
-- 
2.33.1

